lms/djangoapps/grades/services.py

```diff
@@ -49,14 +50,11 @@ def get_subsection_grade_override(self, user_id, course_key_or_id, usage_key_or_
         course_key = _get_key(course_key_or_id, CourseKey)
         usage_key = _get_key(usage_key_or_id, UsageKey)
 
-        grade = self.get_subsection_grade(user_id, course_key, usage_key)
+        # this will raise a PersistentSubsectionGrade.DoesNotExist Exception
```

Comment:
#NIT the comment doesn't really explain why we no longer need the `subsection_grade` object returned

Reply:
Or, maybe clarify this comment to indicate why this call is made here?  For example:

`Verify that a corresponding subsection grade exists with this override.  Raises PersistentSubsectionGrade.DoesNotExist if it doesn't exist.`